config/locales/client.en.yml

```diff
@@ -318,6 +318,7 @@ en:
     undo: "Undo"
     revert: "Revert"
     failed: "Failed"
+    ok: "OK"
```

Comment:
Let's use `Yes/No` instead of `OK` since all the other translations are doing so.